server/paymentProviders/opencollective/manual.js

```diff
@@ -62,7 +62,10 @@ async function processOrder(order) {
   const platformTipEligible = await isPlatformTipEligible(order, host);
   const platformTip = getPlatformTip(order);
   const platformTipInHostCurrency = Math.round(platformTip * hostCurrencyFxRate);
-  const paymentProcessorFeeInHostCurrency = order.data?.paymentProcessorFeeInHostCurrency || 0;
+
+  const paymentProcessorFee = order.data?.paymentProcessorFee || 0;
+  const paymentProcessorFeeInHostCurrency =
+    order.data?.paymentProcessorFeeInHostCurrency || Math.round(paymentProcessorFee * hostCurrencyFxRate) || 0;
```

Comment:
I forgot about this, I don't remember exactly why we introduced it.

https://github.com/opencollective/opencollective-api/pull/6138